dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/publish/files/otel-collector-config.yaml

```diff
@@ -0,0 +1,28 @@
+receivers:
+ otlp:
+   protocols:
+     grpc:
+     http:
+ zipkin:
+
+exporters:
+ logging:
+ jaeger:
+   endpoint: "jaeger-all-in-one:14250"    
+   tls:
+      insecure: true
+ zipkin:
+   endpoint: "http://zipkin-all-in-one:9411/api/v2/spans"
+
+processors:
+ batch:
```

Comment:
It would probably be a good idea to set the batch timeout to 0s here, so that we're not unnecessarily making tests take longer.